lib/compress.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Compressor do
+  @moduledoc """
+  Compress is a small module to compress alpha-numeric strings
+  containing consecutive, duplicate letters.
+  """
+
+  @doc """
+  Compress alpha-numeric string in the format of
+  "aaabbbccc" down to "a3b3c3".
+
+  ## Examples
+
+      iex> Compressor.compress("aaabbbccc")
+      "a3b3c3"
+
+  """
```

Comment:
Nice function docs.

The input and return value are baked in with the example and not explicitly write. Or maybe this style is an Elixir convention?